assets/scss/modules/_phase-banner.scss

```diff
@@ -0,0 +1,8 @@
+.header-banner-wrapper {
+    max-width: 960px;
+    margin: 0 auto;    
+    display: block;
+    @media screen and (max-width: 986px) {
```

Comment:
why is this `986px` ? please use the `@include media()` mixin with a standard size applied